presto-iceberg/src/main/java/com/facebook/presto/iceberg/IcebergConfig.java

```diff
@@ -166,4 +169,17 @@ public boolean isParquetDereferencePushdownEnabled()
     {
         return parquetDereferencePushdownEnabled;
     }
+
+    @Config("iceberg.hive-statistics-merge-type")
+    @ConfigDescription("determines how to merge statistics when configured using a Hive Metastore")
```

Comment:
nit : Can we rename this to `iceberg.hive-statistics-merge-strategy` ? The text could read 'Strategy for merging Hive table statistics (if present) with Iceberg table statistics'

Reply:
fixed